refs/schemas/priv/dictionary/motives/en.motives.json

```diff
@@ -0,0 +1,8 @@
+{
+ "IDENTITY-UNCONFIRMED": "Can't confirm the Data Subject Indentity",
+ "LANGUAGE-UNSUPPORTED": "The language of the request is not understood by the Organization",
+ "LEGAL-GROUNDS": "The System has legal grounds to repsond in the way it responds",
+ "LEGAL-OBLIGATIONS": "The System has legal obligation to repsond in the way it responds",
+ "REQUEST-UNSUPPORTED": "The System does not recongize the particular type of request that is made",
```

Comment:
My idea is that the Privacy Compiler can only respond to requests it supports (and it can only support support requests of which it can recognize the type). 

I don't think it is of interest to the Data Subject to know that a particular System is unable to respond because it does not understand (or because it understands but is not programmed to respond).